src/game/options_menu.c

```diff
@@ -355,6 +357,13 @@ static void optmenu_opt_change(struct Option *opt, s32 val) {
     switch (opt->type) {
         case OPT_TOGGLE:
             *opt->bval = !*opt->bval;
+            if (opt->label == optsVideoStr[5]) {
```

Comment:
That ain't great. Maybe it's better to not modify the flags themselves depending on the `configHUD` variable, but instead to check it in the HUD rendering functions?

Reply:
something in src/game/hud.c at line 453? let me try